### PR TITLE
getting started tutorial

### PR DESCRIPTION
**Pull request by snedamle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
 table, insert records into it, and select records from the table. This also shows how persisted records can be retrieved by
 the enclave once the host is re-started.
 
-### Initial SetUp
+# Write your first Conclave application in 5 mins
 
-In order to compile, and run these samples successfully, kindly download the [conclave SDK](https://www.conclave.net/get-conclave/) and update 
-the conclaveRepo property in gradle.properties to point to this path.
+This tutorial can be used to get started with Conclave in 5 mins. We will create a sample in which the client can send
+a string to the enclave, the enclave will reverse it and send it back to the client.
```
Comment (gendal) [inline README.md]: "... and the machine on which the enclave is running will not be able to discover what the string was, even though the reversal logic is executing on that machine!" (assuming we show them how to do release mode?)

File: README.md [hunk 2]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
...
 
-    conclaveRepo=add_path_to_repo
+#Step 1: Prerequisites
+You need [JDK](https://www.openlogic.com/openjdk-downloads) 8 or 11 installed in order to build and run the app.<br />
```
Comment (gendal) [inline README.md]: this is a strange URI.  Can we point to a more standard or non-commercial JDK?

File: README.md [hunk 3]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
...
+Grab a copy of Conclave-SDK from [conclave.net](https://www.conclave.net/get-conclave/)
+We will use the conclave init tool for bootstrapping Conclave projects, reducing the amount of boilerplate you need to write.
```
Comment (gendal) [inline README.md]: should we put conclave init in code-style formatting?  `conclave init` (or whatever the markdown is)

File: README.md [hunk 4]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
...
+# Step 2: Create the project template using the conclave-init tool
+
+To create a new Conclave project, run the following command:
+
+    java -jar /path/to/conclave-sdk/tools/conclave-init.jar 
+    --package com.megacorp 
+    --enclave-class-name ReverseEnclave 
+    --target ./reverse-conclave-app
+
+A new project would have been created in the reverse-conclave-app directory, as specified by --target ./reverse-conclave-app.
```
Comment (gendal) [inline README.md:70]: does it create the directory if it doesn't already exist?

File: README.md [hunk 5]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
...
+1. The enclave module contains the ReverseEnclave class. Your enclave code will go here.
+2. The host directory has been created, too.
+   It contains only a build.gradle file, since that's all that's required for the [Conclave web host](https://docs.conclave.net/conclave-web-host.html).
+3. The client directory contains a basic client which can interact with the web host.
+
+To understand more about the different files generated by this tool jump to the below section named Configurations generated by the conclave-init tool.
+
+# Step 3: Add the Enclave code
+
+Copy paste the below code to receiveMail method in SimpleEnclave class. This logic reverses the string received by the client and sends it back to the client.
```
Comment (gendal) [inline README.md]: Copy and paste the code below into the 'receiveMail' method in the SimpleEnclave class.

File: README.md [hunk 6]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
...
+
+# Step 4: Run this sample
+
+There are different [modes](https://docs.conclave.net/enclave-modes.html) in which you can run your Conclave application.
+For this tutorial we will run the conclave application in mock mode. To learn more about the different modes, please
+read the below section named Enclave Modes.
+
+1. Build the host
+
+        ./gradlew clean host:shadowJar -PenclaveMode=mock
```
Comment (gendal) [inline README.md]: do we need to tell the reader where to run this from?

File: README.md [hunk 7]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
...
+
+2. Build the client
+
+        ./gradlew client:shadowJar
+
+3. Run the host
+
+        java -jar host/build/libs/host-mock.jar
+
+5. Run the client
```
Comment (gendal) [inline README.md]: this will need to be from a different terminal window i guess as the host will be blocking the previous one?

File: README.md [hunk 8]
```diff
@@ -46,12 +46,274 @@ setting up a database inside an enclave is very well possible. This sample shows
...
+
+        java -jar client/build/libs/client-all.jar "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" ReverseMe
+
+Using above 4 steps you should be able to have a conclave application running on your system.
```
Comment (gendal) [inline README.md:116]: Maybe add a final line such as: "Now you have developed and tested your enclave, you can take a final step, to achieve the privacy promise of Conclave. To do this, you can run the enclave in 'RELEASE' mode. See here.

**Review by gendal — COMMENTED** [review]
Thanks Sneha. Nicely simple!  I've not reviewed the appendix - let me know if you'd like me to

File: README.md (hunk 1 above)
Comment (snedamle) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (snedamle) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (snedamle) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (snedamle) [reply README.md:70]: yes, it creates the directory if it doesn't exist.

File: README.md (hunk 5 above)
Comment (snedamle) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (snedamle) [reply README.md]: yes, updated this.

File: README.md (hunk 7 above)
Comment (snedamle) [reply README.md]: mentioned the details

File: README.md (hunk 8 above)
Comment (snedamle) [reply README.md:116]: done

**Pull request merged**